frame/quorum/src/tests.rs

```diff
@@ -21,21 +21,39 @@ use crate::{
   },
   pallet::*,
 };
-use frame_support::{assert_ok, traits::Hooks, BoundedVec};
+use frame_support::{assert_noop, assert_ok, traits::Hooks, BoundedVec};
 use tidefi_primitives::{
   pallet::SecurityExt, ComplianceLevel, CurrencyId, Hash, Mint, ProposalType,
 };
 
+struct Context {
```

Comment:
Super! This is really something we should do to all our tests, maybe we could look to create a share library to reduce our tests code base!

Well done ;)

Reply:
That's a good plan. :)